### ISSUE TITLE
Option to show current view name in place of Lovelace title

### ISSUE THREAD

**Issue by SeanPM5** [home-assistant/home-assistant-polymer#4787]
## The request

At the top of your screen is the "Title of your Lovelace UI" 

![default](https://i.imgur.com/f17H1AG.png)

I think a cool **option** might be to display the active view name in place of that. 

![view](https://i.imgur.com/6FJsI9F.png)

When you click the pencil icon next to the title, underneath the text field to rename it, would be a toggle. When this is toggled on it, it would display the text field in a disabled / grayed out state.

![current-view](https://i.imgur.com/qobeIbC.jpg)

**Comment by zsarnett** [comment]
Would it be better to append the view name here?

`Home - Lights View`

**Comment by danimart1991** [comment]
I would like to take this Issue.

The idea is to add an option to Title Editor: "Append selected view Title to Title" or something like that, and then, like @zsarnett said, append the view name to the Title.

I appreciate info about translation and the string to put in the option.

**Comment by Mariusthvdb** [comment]
Yes, the append sounds great because that would allow easier distinction between different instances.

Have the instance name appended with the view title. At least as option please, for those that have more than 1 instance (using 3 here and heavily relying on that naming convention, which now is possible using CustomHeader and FavIcon)

**Comment by danimart1991** [comment]
I would like to take this issue.

**Comment by danimart1991** [comment]
According to the comments in the Issue.

What do you think about a first approach appending the view name to the title automatically if more than 3 views configured?
